tests/test_other.py

```diff
@@ -2555,6 +2555,43 @@ def test_module_exports_with_closure(self):
     try_delete(path_from_root('tests', 'Module-exports', 'test.js.map'))
     try_delete(path_from_root('tests', 'Module-exports', 'test.js.mem'))
 
+  def test_node_catch_exit(self):
+    # Test that in node.js exceptions are not caught if NODEJS_EXIT_CACTH=0
+    if not NODE_JS in JS_ENGINES:
+        return
+
+    open(os.path.join(self.get_dir(), 'count.c'), 'w').write('''
+      #include <string.h>
+      int count(const char *str) {
+          return (int)strlen(str);
+      }
+    ''')
+
+    open(os.path.join(self.get_dir(), 'index.js'), 'w').write('''
+      const count = require('./count.js');
+
+      console.log(xxx); //< here is the ReferenceError
+    ''')
+
+    Popen([PYTHON, EMCC, os.path.join(self.get_dir(), 'count.c'), '-o', 'count.js']).communicate()
+
+    # Check that compilation was successful
+    assert os.path.exists(os.path.join(self.get_dir(), 'count.js'))
+
+    # Check that the ReferenceError is caught and rethrown
+    self.assertNotContained('console.log(xxx); //< here is the ReferenceError',
```

Comment:
please save this string to a variable, and reuse it below - to avoid risks of a typo making the test invalid